Baconit/Strings/es-es/Resources.resw

```diff
@@ -0,0 +1,889 @@
+    <value>Ordenar:</value>
+  </data>
+  <data name="SortPostTB.Text" xml:space="preserve">
+    <value>Ordenar:</value>
+  </data>
+  <data name="SortTextHotTB.Text" xml:space="preserve">
+    <value>Temas candentes</value>
+  </data>
+  <data name="SortTimeTB.Text" xml:space="preserve">
+    <value>La última semana</value>
+  </data>
+  <data name="StickyTB.Text" xml:space="preserve">
+    <value>Pegados</value>
+  </data>
+  <data name="StickyUserTB.Text" xml:space="preserve">
+    <value>Pegado</value>
+  </data>
+  <data name="SubtmitPostTitleTB.Text" xml:space="preserve">
+    <value>Título</value>
+  </data>
+  <data name="TimeTB.Text" xml:space="preserve">
+    <value>Hora:</value>
+  </data>
+  <data name="ViewContextTB.Text" xml:space="preserve">
+    <value>Ver en contexto</value>
+  </data>
+  <data name="CLinkFMI.Text" xml:space="preserve">
+    <value>Copiar enlace</value>
+  </data>
+  <data name="CMBest.Text" xml:space="preserve">
+    <value>Lo mejor</value>
+  </data>
+  <data name="CMContro.Text" xml:space="preserve">
+    <value>Controvérsico</value>
+  </data>
+  <data name="CMNew.Text" xml:space="preserve">
+    <value>Nuevo</value>
+  </data>
+  <data name="CMOld.Text" xml:space="preserve">
+    <value>Viejo</value>
+  </data>
+  <data name="CMQA.Text" xml:space="preserve">
+    <value>Q&amp;A</value>
+  </data>
+  <data name="CMTop.Text" xml:space="preserve">
+    <value>Lo mejor</value>
+  </data>
+  <data name="collapseTBN.Text" xml:space="preserve">
+    <value>Unir</value>
+  </data>
+  <data name="CopyPerFMI.Text" xml:space="preserve">
+    <value>Copiar el enlace</value>
+  </data>
+  <data name="CountTB.Text" xml:space="preserve">
+    <value>Contar:</value>
+  </data>
+  <data name="CPermaFMI.Text" xml:space="preserve">
+    <value>Copiar el enlace</value>
+  </data>
+  <data name="DeletePostClick.Text" xml:space="preserve">
+    <value>Eliminar el post</value>
+  </data>
+  <data name="EditPostClick.Text" xml:space="preserve">
+    <value>Editar el post</value>
+  </data>
+  <data name="MoreTB.Text" xml:space="preserve">
+    <value>Más</value>
+  </data>
+  <data name="RunGilded.Text" xml:space="preserve">
+    <value>Exclusiva</value>
+  </data>
+  <data name="RunSticky.Text" xml:space="preserve">
+    <value>Pegado</value>
+  </data>
+  <data name="ShareCommentFMI.Text" xml:space="preserve">
+    <value>Compartir comentario</value>
+  </data>
+  <data name="SharePostFMI.Text" xml:space="preserve">
+    <value>Compartir post</value>
+  </data>
+  <data name="TapViewEntire.Text" xml:space="preserve">
+    <value>Pulsa en aquí para ver el hilo completo</value>
+  </data>
+  <data name="AboutDescription.Text" xml:space="preserve">
+    <value>Reddunt es un maravilloso cliente de Reddit basado en Baconit, una aplicación de código libre. Está disponible para todos los dispositivos con Windows 10. 
+El principal objetivo es la optimización y la utilización de características propias de Windows 10!</value>
```

Comment:
I think it could be better if you change "la utilización" for "el uso".

Reply:
I'm used to use both of them as synonymous. I'll keep your preference.